File: common/src/main/java/me/pandamods/pandalib/core/client/event/EventHandlerClient.java
package me.pandamods.pandalib.core.client.event;

import me.pandamods.pandalib.core.network.ConfigNetworking;
import net.minecraft.client.player.LocalPlayer;

public class EventHandlerClient {
	public static void init() {
		// TODO: Implement player join event
//		ClientPlayerEvent.CLIENT_PLAYER_JOIN.register(EventHandlerClient::onClientPlayerJoin);
	}

	private static void onClientPlayerJoin(LocalPlayer localPlayer) {
		ConfigNetworking.SyncClientConfigs();
	}
}

File: common/src/main/kotlin/me/pandamods/pandalib/core/client/event/EventHandlerClient.kt
package me.pandamods.pandalib.core.client.event

import dev.architectury.event.events.client.ClientPlayerEvent
import me.pandamods.pandalib.core.network.ConfigNetworking
import net.fabricmc.api.EnvType
import net.fabricmc.api.Environment
import net.minecraft.client.player.LocalPlayer

@Environment(EnvType.CLIENT)
object EventHandlerClient {
	fun init() {
		ClientPlayerEvent.CLIENT_PLAYER_JOIN.register(ClientPlayerEvent.ClientPlayerJoin { obj: LocalPlayer -> onClientPlayerJoin(obj) })
	}

	private fun onClientPlayerJoin(localPlayer: LocalPlayer) {
		ConfigNetworking.SyncClientConfigs()
	}
}
